### PR TITLE
Select Box PR

### PR DESCRIPTION
**Pull request by VWIJECH**
Changers were added  to the script as mentioned in following document.
[https://docs.google.com/spreadsheets/d/1BfD5O0W6_-t7PgcrtlD6_sO06bBr9JXAAiwyLxanosg/edit#gid=915380837](url)

All tests pass without issue .

### CODE REVIEW

File: src/main/java/elements/fixtures/inputs.html [hunk 1]
```diff
@@ -1,130 +1,167 @@
...
-</head>
-
-<body>
- 	<form action="demo_form.asp">
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+<title>Inputs</title>
+<link rel="stylesheet"	href="http://laxapana.elements.surge.sh/elements_selectbox_bug.css" />
```
Comment (umahaea) [inline src/main/java/elements/fixtures/inputs.html]: If the elements code is already merged to "V0", go ahead and point this to elements.css. Refer one of the fixtures.

**Comment by umahaea** [comment]
@VWIJECH - some of the test names are not suffixed with "Test". Please do it. And please format the code well.

**Comment by VWIJECH** [comment]
Added requested changers.

**Comment by umahaea** [comment]
Pull Request?

On Wed, May 25, 2016 at 1:35 PM, VWIJECH <notifications@github.com> wrote:

> Added requested changers.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/Pearson-Higher-Ed/ux-test-platform/pull/46#issuecomment-221682835>
>



-- 
Software Quality Assurance Engineer
Team - Pearson Design Accelerator
Centennial CO
14087025616

**Pull request closed without merging**